packages/components/src/composite/legacy/stories/index.story.tsx

```diff
@@ -15,7 +15,8 @@ import {
 import { UseCompositeStatePlaceholder, transform } from './utils';
 
 const meta: Meta< typeof UseCompositeStatePlaceholder > = {
-	title: 'Components/Composite',
+	// TODO: should we keep this story around? If so, how should we call it?
+	title: 'Components/Composite/Legacy',
```

Comment:
Probably @mirka is best-suited for the above TODO question. Also, see other comment on the current implementation's Storybook `title`

Reply:
Maybe we should keep it like we keep other deprecated component stories (like `Navigation`)?

Reply:
Yes, I think we should keep it as long as it's available in the package. As for how, my recommendation would be like this:

```diff
diff --git a/packages/components/src/composite/legacy/stories/index.story.tsx b/packages/components/src/composite/legacy/stories/index.story.tsx
index 08eebc65ed..d3d3205afe 100644
--- a/packages/components/src/composite/legacy/stories/index.story.tsx
+++ b/packages/components/src/composite/legacy/stories/index.story.tsx
@@ -15,8 +15,8 @@ import {
 import { UseCompositeStatePlaceholder, transform } from './utils';
 
 const meta: Meta< typeof UseCompositeStatePlaceholder > = {
-	// TODO: should we keep this story around? If so, how should we call it?
-	title: 'Components/Composite/Legacy',
+	title: 'Components (Deprecated)/Composite (Unstable)',
+	id: 'components-composite-unstable',
 	component: UseCompositeStatePlaceholder,
 	subcomponents: {
 		// @ts-expect-error Storybook doesn't like overloaded exports as subcomponents
diff --git a/packages/components/src/composite/legacy/stories/utils.tsx b/packages/components/src/composite/legacy/stories/utils.tsx
index 06edd34863..2fb51c845f 100644
--- a/packages/components/src/composite/legacy/stories/utils.tsx
+++ b/packages/components/src/composite/legacy/stories/utils.tsx
@@ -8,6 +8,25 @@ import type { StoryContext } from '@storybook/react';
  */
 import type { LegacyStateOptions } from '..';
 
+/**
+ * Renders a composite widget.
+ *
+ * This unstable component is deprecated. Use `Composite` instead.
+ *
+ * ```jsx
+ * import {
+ * 	__unstableUseCompositeState as useCompositeState,
+ * 	__unstableComposite as Composite,
+ * 	__unstableCompositeItem as CompositeItem,
+ * } from '@wordpress/components';
+ *
+ * const state = useCompositeState();
+ * <Composite state={ state }>
+ * 	<CompositeItem>Item 1</CompositeItem>
+ * 	<CompositeItem>Item 2</CompositeItem>
+ * </Composite>;
+ * ```
+ */
 export function UseCompositeStatePlaceholder( props: LegacyStateOptions ) {
 	return (
 		<dl>

```

The important points being:

- It needs to be clear that we're referring to the `__unstable` version, and that it's deprecated.
- Usually we want to ensure that we [don't break permalinks](https://github.com/WordPress/gutenberg/blob/trunk/storybook/manager-head.html#L2), but in this specific case I think we would cause the less damage if we just made a clean break and changed the URLs so it reflects the new situation.

Reply:
Sounds good, done in https://github.com/WordPress/gutenberg/pull/63564/commits/b3e58bc5c1be96c27f8535ed7e6958da5ced2555